make check fails because of an isort's breaking change

## Description of the bug

GitHub Actions failed https://github.com/HazyResearch/fonduer/runs/839099081 because isort's breaking change.

isort has been updated 7/4 (https://github.com/timothycrosley/isort/blob/master/CHANGELOG.md#500-penny---july-4-2020)
v5.0.0 introduced many breaking changes and the following change broke fonduer's testing code.
- --recursive option has been removed. Directories passed in are now automatically sorted recursive.

## To Reproduce
Steps to reproduce the behavior:
1. Update isort v5.0.0+
2. Execute `make check`

## Expected behavior

Tests pass.

## Error Logs/Screenshots

```
isort -rc -c src/
usage: isort [-h] [--src SRC_PATHS] [-a ADD_IMPORTS] [--ac] [--af]
             [-b KNOWN_STANDARD_LIBRARY]
             [--extra-builtin EXTRA_STANDARD_LIBRARY] [-c] [--ca] [--cs] [-d]
             [--df] [--ds] [-e] [-f KNOWN_FUTURE_LIBRARY] [--fas] [--fass]
             [--ff FROM_FIRST] [--fgw [FORCE_GRID_WRAP]] [--fss] [-i INDENT]
             [-j JOBS] [-k] [--lai LINES_AFTER_IMPORTS]
             [--lbt LINES_BETWEEN_TYPES] [--le LINE_ENDING] [--ls]
             [-m {GRID,VERTICAL,HANGING_INDENT,VERTICAL_HANGING_INDENT,VERTICAL_GRID,VERTICAL_GRID_GROUPED,VERTICAL_GRID_GROUPED_NO_COMMA,NOQA,VERTICAL_HANGING_INDENT_BRACKET,VERTICAL_PREFIX_FROM_MODULE_IMPORT,0,1,2,3,4,5,6,7,8,9}]
             [-n] [--nis] [--nlb NO_LINES_BEFORE] [-o KNOWN_THIRD_PARTY]
             [--ot] [--dt] [-p KNOWN_FIRST_PARTY] [-q] [--rm REMOVE_IMPORTS]
             [--rr] [-s SKIP] [--sd DEFAULT_SECTION] [--sg SKIP_GLOB] [--sl]
             [--nsl SINGLE_LINE_EXCLUSIONS] [--sp SETTINGS_PATH]
             [-t FORCE_TO_TOP] [--tc] [--up] [-V] [-v]
             [--virtual-env VIRTUAL_ENV] [--conda-env CONDA_ENV] [--vn]
             [-l LINE_LENGTH] [--wl WRAP_LENGTH] [--ws] [--case-sensitive]
             [--filter-files] [--py {all,2,27,3,35,36,37,38,39,auto}]
             [--profile {black,django,pycharm,google,open_stack,plone,attrs,hug}]
             [--interactive] [--old-finders] [--show-config]
             [files [files ...]]
isort: error: unrecognized arguments: -rc
Makefile:19: recipe for target 'check' failed
make: *** [check] Error 2
##[error]Process completed with exit code 2.
```

## Environment (please complete the following information)
 - isort: v5.0.0

## Additional context
Add any other context about the problem here.

--- a/.isort.cfg
+++ b/.isort.cfg
@@ -1,6 +1,5 @@
 [settings]
 multi_line_output=3
-not_skip = __init__.py
 include_trailing_comma=True
 force_grid_wrap=0
 combine_as_imports=True

--- a/Makefile
+++ b/Makefile
@@ -16,8 +16,8 @@ test: dev check docs
 	pytest tests
 
 check:
-	isort -rc -c src/
-	isort -rc -c tests/
+	isort -c src/
+	isort -c tests/
 	black src/ --check
 	black tests/ --check
 	flake8 src/

--- a/requirements-dev.txt
+++ b/requirements-dev.txt
@@ -3,7 +3,7 @@ flake8>=3.8.0
 flake8-docstrings
 mypy
 nltk
-isort
+isort>=5.0.0
 pre-commit
 pytest
 pytest-dependency

--- a/src/fonduer/features/feature_libs/tree_structs.py
+++ b/src/fonduer/features/feature_libs/tree_structs.py
@@ -3,7 +3,7 @@ import re
 from functools import lru_cache
 from typing import Any, Dict, List, Optional, Union
 
-import lxml.etree as et
+from lxml import etree as et
 from lxml.etree import _Element
 
 from fonduer.parser.models import Sentence

--- a/src/fonduer/learning/modules/concat_linear.py
+++ b/src/fonduer/learning/modules/concat_linear.py
@@ -2,8 +2,7 @@
 from typing import Any, Dict, List
 
 import torch
-import torch.nn as nn
-from torch import Tensor
+from torch import Tensor, nn as nn
 
 
 class ConcatLinear(nn.Module):

--- a/src/fonduer/learning/modules/soft_cross_entropy_loss.py
+++ b/src/fonduer/learning/modules/soft_cross_entropy_loss.py
@@ -2,9 +2,8 @@
 from typing import List
 
 import torch
-import torch.nn as nn
-import torch.nn.functional as F
-from torch import Tensor
+from torch import Tensor, nn as nn
+from torch.nn import functional as F
 
 
 class SoftCrossEntropyLoss(nn.Module):

--- a/src/fonduer/learning/task.py
+++ b/src/fonduer/learning/task.py
@@ -3,14 +3,13 @@ import logging
 from functools import partial
 from typing import Any, Dict, List, Optional, Union
 
-import torch.nn as nn
-import torch.nn.functional as F
 from emmental.modules.embedding_module import EmbeddingModule
 from emmental.modules.rnn_module import RNN
 from emmental.modules.sparse_linear_module import SparseLinear
 from emmental.scorer import Scorer
 from emmental.task import EmmentalTask
-from torch import Tensor
+from torch import Tensor, nn as nn
+from torch.nn import functional as F
 
 from fonduer.learning.modules.concat_linear import ConcatLinear
 from fonduer.learning.modules.soft_cross_entropy_loss import SoftCrossEntropyLoss

--- a/src/fonduer/packaging/fonduer_model.py
+++ b/src/fonduer/packaging/fonduer_model.py
@@ -352,6 +352,7 @@ def _get_default_conda_env() -> Optional[Dict[str, Any]]:
              :func:`save_model()` and :func:`log_model()`.
     """
     import torch
+
     import fonduer
 
     return _mlflow_conda_env(

--- a/src/fonduer/parser/lingual_parser/spacy_parser.py
+++ b/src/fonduer/parser/lingual_parser/spacy_parser.py
@@ -15,8 +15,8 @@ from fonduer.parser.models.sentence import Sentence
 
 try:
     import spacy
-    from spacy.cli import download
     from spacy import util
+    from spacy.cli import download
     from spacy.tokens import Doc
 except Exception:
     raise Exception("spaCy not installed. Use `pip install spacy`.")

